mcp/prow-analyzer/setup_venv.sh

```
@@ -0,0 +1,54 @@
+#
+# WHY THIS IS NEEDED:
+# When running from Cursor's integrated terminal, the environment is polluted with
+# AppImage-specific variables (APPIMAGE, APPDIR, LD_LIBRARY_PATH, etc.) and PATH
+# entries pointing to the mounted AppImage. This causes Python's venv module to
+# incorrectly record the AppImage as the Python interpreter's home, resulting in
+# symlinks like venv/bin/python -> cursor.appimage instead of the system Python.
+#
+# This script cleans the environment before creating the venv to ensure proper
+# Python interpreter references.
+
+cd "$(dirname "$0")"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)